Cargo.toml

```diff
@@ -0,0 +1,17 @@
+[workspace]
+resolver = "2"
+members = ["formatting"]
+
+[workspace.package]
+version = "0.1.0"
+authors = ["Miden Developers"]
+repository = "https://github.com/0xPolygonMiden/miden-formatting"
+homepage = "https://github.com/0xPolygonMiden/miden-formatting"
+documentation = "https://github.com/0xPolygonMiden/miden-formatting"
+categories = ["value-formatting"]
+keywords = ["pretty-print", "prettier", "miden"]
+license-file = "LICENSE"
+readme = "README.md"
+edition = "2021"
```

Comment:
A few small nits:

1. For the `authors` field we usually say "miden contributors` (e.g., [here](https://github.com/0xPolygonMiden/miden-vm/blob/main/miden/Cargo.toml#L5)).
2. For the `documentation` field we usually leave a link to docs.rs (e.g., [here](https://github.com/0xPolygonMiden/miden-vm/blob/main/miden/Cargo.toml#L9)) - so, it might need to be in the crate-specific `Cargo.toml`.
3. Instead of the `license-file`, we usually have `license` field set to "MIT".
4. We also usually add `rust-version` field (either 1.75 or 1.76 is fine).

Reply:
Sure thing, can make these changes first thing tomorrow.